app/experimenter/experiments/views.py

```diff
@@ -25,6 +26,10 @@
 
 
 class ExperimentFiltersetForm(forms.ModelForm):
+    show_experiment_changes = forms.ChoiceField(required=False)
```

Comment:
So I put this in here because that's what I thought you had to do to be able to get it into the `fields` section in the `Meta` class on the form... which is what appears to be determining the order of the filters in the whole filter area. I learned this from the search text area PR #1187. But now that you point this out.. I'm realizing that I don't care about this because right now all the date stuff lives at the bottom of the search area anyway.. and if I comment all this out, it works just fine. I think I could do away with this stuff, and the corresponding lines in the `fields` section. I have to admit it was always magic to me why this worked for the search text area I added in PR #1187 .

Reply:
Hmm. I'm looking through the django-filter code now and I see https://github.com/carltongibson/django-filter/blob/24adad8c48bc9e7c7539b6510ffde4ce4effdc29/django_filters/filterset.py#L332-L334 -- maybe it will give you a warning if you comment it out?

Reply:
I actually haven't seen any warnings with this stuff taken out. I am going to leave it out since it apparently doesn't do anything.